### PR TITLE
Python2/3 compatibility in integration test 647 and mock services

### LINKED ISSUE #3173: Port authz helper test script to Python 3
This should resurrect test 647 (bearer authorization) on EL9.

### PR DESCRIPTION
**Pull request by vvolkl**
Fixes #3173

As far as I can tell, `cvmfs-scitokens-helper.py` is an older duplicate of `cvmfs_scitoken_helper` and not used anywhere, so I remove it.

### CODE REVIEW

**Comment by vvolkl** [comment]
@cernvm-bot cloudtest

**Review by jblomer — CHANGES_REQUESTED** [review]
Nice!  So let's also reenable the 647 test in test/cloud_testing/platforms/centos9_x86_64_test.sh

**Comment by jblomer** [comment]
@cernvm-bot cloudtest

**Comment by jblomer** [comment]
Hm, there seems to be still a problem, now both on EL8 and 9. On EL7, the test passes.

**Comment by vvolkl** [comment]
Yes, I'm having problems getting this test to run locally even on EL7, will have to continue investigating.

**Comment by vvolkl** [comment]
@cernvm-bot cloudtest

File: test/test_functions [hunk 1]
```diff
@@ -836,7 +836,7 @@ create_filled_repo() {
 # @param  repo_dir   where to put the dummy dirents
 make_huge_repo() {
   local repo_dir=$1
-  $CVMFS_PYTHON2 ${TEST_ROOT}/common/mock_services/make_repo.py \
+  ${TEST_ROOT}/common/mock_services/make_repo.py                \
```
Comment (HereThereBeDragons) [inline test/test_functions:839]: you could also use `$CVMFS_SYS_PYTHON` as python selector

Comment (vvolkl) [reply test/test_functions:839]: Hmm, what's the advantage of `$CVMFS_SYS_PYTHON` over `#!/usr/bin/env python`?

**Comment by vvolkl** [comment]
@cernvm-bot cloudtest

**Comment by jblomer** [comment]
@cernvm-bot help

**Comment by vvolkl** [comment]
@cernvm-bot cloudtest cc7 cc8

**Comment by vvolkl** [comment]
@cernvm-bot  cloudtest cc8

**Comment by vvolkl** [comment]
@cernvm-bot cloudtest cc8

**Comment by vvolkl** [comment]
@cernvm-bot cloudtest cc8

**Comment by vvolkl** [comment]
@cernvmbot cloudtest cc8 nodestroy

**Comment by vvolkl** [comment]
@cernvm-bot cloudtest cc8 nodestroy

**Comment by vvolkl** [comment]
@cernvm-bot cloudtest cc8

**Comment by vvolkl** [comment]
@cernvm-bot cloudtest

**Comment by vvolkl** [comment]
@cernvm-bot cloudtest

**Comment by vvolkl** [comment]
@cernvm-bot cloudtest

**Pull request merged**